docs/gatsby-config.js

```diff
@@ -16,30 +16,38 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            'graphql-playground',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'schema/schema',
+            'schema/scalars-enums',
+            'schema/unions-interfaces',
+            'schema/directives',
+            'schema/creating-directives',
           ],
+          'Serving Requests': [
+            'serving/data',
+            'serving/data-sources',
+            'serving/errors',
+            'serving/file-uploads',
+            'serving/subscriptions',
+          ],
+          Testing: ['testing/mocking', 'testing/testing'],
+          Performance: ['performance/caching', 'performance/apq'],
+          Security: ['security/authentication', 'security/terminating-ssl'],
+          Integrations: ['integrations/middleware'],
+          Deployment: [
+            'deployment/heroku',
+            'deployment/lambda',
+            'deployment/now',
+            'deployment/netlify',
+            'deployment/azure-functions',
+          ],
+          Monitoring: ['monitoring/metrics', 'monitoring/health-checks'],
           Federation: [
```

Comment:
Can we move federation to a more prominent spot? Possibly above testing?

Reply:
Yep! This ordering is largely attempting to mirror the order in which someone might encounter these features, but I reckon Federation is an important-enough piece of the puzzle that it merits some exceptional treatment :D